mne/time_frequency/tfr.py

```diff
@@ -605,36 +960,12 @@ def __init__(self, info, data, times, freqs, nave, comment=None,
             raise ValueError("Number of times and data size don't match"
                              " (%d != %d)." % (n_times, len(times)))
         self.data = data
-        self.times = np.asarray(times)
-        self.freqs = np.asarray(freqs)
+        self.times = np.array(times)
```

Comment:
and it should probably be `np.array(times, float)` to guarantee we get the right dtype